### DISCUSSION TITLE
get listing by shop giving 0 count and empty array

### DISCUSSION THREAD

**Discussion by slashcartshop** [etsy/open-api#1216] [Q&A]
i have one listing in my shop but get listing by shop giving 0 count and empty array in response

**Comment by Gareth064** [comment]
What is the URL you are calling and can you share the URL to the shop so we can look at the listing?

**Reply by slashcartshop** [reply]
@Gareth064 
Thanks for your reply

i am using this api `{{baseUrl}}/v3/application/shops/{{shopID}}/listings?limit=25&offset=0&sort_on=created&sort_order=desc&includes=&client_id={{client_id}}`
and it responses
{
    "count": 0,
    "results": []
}

`_When i use {{baseUrl}}/v3/application/listings/{{listingId}}?includes=&language=&client_id={{client_id}} then it show listing in resposne_`

**shop url**: https://www.etsy.com/uk/shop/SlashCart
but i have there listing products which are active in dashboard

**Reply by slashcartshop** [reply]
@Gareth064 

when i use `{{baseUrl}}/v3/application/shops/{{shopID}}/listings?state=inactive&limit=25&offset=0&sort_on=created&sort_order=desc&includes=&client_id={{client_id}}` then **In Active** listing fetch but when i use `status=active` then it not fetch active listing

**Reply by Gareth064** [reply]
Looks like your shop is in holiday mode. Pretty sure this stops listings from being returned. Try coming off that to see what happens.
